### ISSUE TITLE
minimalcss won't return any output if javascript breaks

### ISSUE THREAD

**Issue by ggiak** [peterbe/minimalcss#399]
Hi, @peterbe,

I am running minimalcss in both command line and in node using the following config:
```
   ignoreJSErrors: true,
   ignoreCSSErrors: true,
   withoutJavascript: true,
   ignoreRequestErrors: true,
   puppeteerArgs: ['--no-sandbox']
```

in case the page is having a JS error while rendering, instead of the minimal css it throws an exception.

Same this at minimalcss.app website.

check the following screenshot:
![image](https://user-images.githubusercontent.com/20743694/93715343-6144da80-fb71-11ea-80dc-ed1ce90b2b30.png)

is there a config to avoid this ?

**Comment by peterbe** [comment]
Clearly, the website (which is ultimately just a demo) doesn't expose the option but you can perhaps use: `ignoreRequestErrors: true` as an option. It basically monitors any responses and if their response code is `>=400` it just ignores that URL.

**Comment by ggiak** [comment]
> Clearly, the website (which is ultimately just a demo) doesn't expose the option but you can perhaps use: `ignoreRequestErrors: true` as an option. It basically monitors any responses and if their response code is `>=400` it just ignores that URL.

Hi @peterbe,

thank you for your reply.
I have tried both the website + minimalcss through nodejs.

the nodejs config I have used is:

```
ignoreJSErrors: true,
ignoreCSSErrors: true,
withoutJavascript: true,
**ignoreRequestErrors: true,**
puppeteerArgs: ['--no-sandbox']

```

and the error I am getting is:
`Error Happened Error: 404 on https://tagmanager.smartadserver.com/3433/362707/smart.prebid.js`

**Comment by ggiak** [comment]
@peterbe, looks like this commit:

https://github.com/peterbe/minimalcss/blob/719ed2c063605cf51a7089a27ae14e3e935d4c78/src/run.js

doesn't exist on v.0.10.0.

other than that, I tested against master and it looks like it is working as expected.

**Comment by peterbe** [comment]
So, I guess we need a new release. I'm on it. Today.

**Comment by ggiak** [comment]
@peterbe 
Let me know if you need me to test any more features from new-committed features that doesn't exist in tag v0.10.0

**Comment by peterbe** [comment]
Try 0.11.0 now. Some other minor changes in there.

**Comment by ggiak** [comment]
> Try 0.11.0 now. Some other minor changes in there.

Works as charm now 👍 💯 🥇

**Comment by ggiak** [comment]
Closed by #197
